web/src/main/java/org/openmrs/web/Listener.java

```diff
@@ -230,10 +230,7 @@ public void contextInitialized(ServletContextEvent event) {
 				if (StringUtils.hasLength(appDataRuntimeProperty)) {
 					OpenmrsUtil.setApplicationDataDirectory(null);
 				}
-				//ensure that we always log the runtime properties file that we are using
-				//since openmrs is just booting, the log levels are not yet set. TRUNK-4835
-				OpenmrsLoggingUtil.applyLogLevel(getClass().getName(), "INFO");
-				log.info("Using runtime properties file: {}",
+				log.warn("Using runtime properties file: {}",
```

Comment:
I guess this technically isn't a "warn" but this is much better solution IMO if we want to force this to be logged than (unintentionally) setting all of "org.openmrs" to INFO from this point forward....